### ISSUE TITLE
[generic.reset] Undo normalize's font-family on buttons, inputs, ...

### ISSUE THREAD

**Issue by lhermann** [inuitcss/inuitcss#362]
Hi good folks of inuitcss.

I discovered that normalize.css does the following with buttons, inputs, ...

```css
/**
 * 1. Change the font styles in all browsers (opinionated).
 */
button,
input,
optgroup,
select,
textarea {
  font-family: sans-serif; /* 1 */
  ...
}
```

This, however, overwrites a font-family defined in `html` or `body`. Therefore I propose to add

```css
button,
input,
optgroup,
select,
textarea {
    font-family: inherit;
}
```

in `_generic.reset.scss` to "undo" this opinionated setting and inherit the font-family from `html` or `body`. I can create a PR if you are fine with it.

**Comment by nemanjaglumac** [comment]
Hi @lhermann ,
before someone from core members responds, just my quick question:

If you define those rules in Elements layer (for example `_elements.typography.scss), wouldn't it automatically override normalize.css (since it comes later on)?

**Comment by lhermann** [comment]
@nemanjaglumac `_elements.typography.scss` is exactly where I defined it, like so:
```scss
html {
    font-family: $font-primary;
    ...
}
```
[I made a little codepen](https://codepen.io/lhermann/pen/zjGqBo) to test it. Result: normalise.css does overwrite `font-family` on button, select, ... if it is defined in an `html` selector, but not if it is defined in a `*` selector.

**Comment by nemanjaglumac** [comment]
@lhermann that shouldn't happen. How does your main.scss file look like? Order of the imports is important and `elements` should come *after* generic layer.

Ok, you have set font-family for the html, but do you also have your originally proposed 'reset' in that `_elements.typography.scss`? 

```
html {
    font-family: $font-primary;
}

button,
input,
optgroup,
select,
textarea {
    font-family: inherit;
}
```

If that's included and the order of imports is correct, it would be very strange if normalize.css still overrides those settings.

**Comment by lhermann** [comment]
Yes, of course, you are correct. And that's exactly how I solved it locally. BUT I thought this proposed 'reset' might be a candidate to add to `generic.reset` since it carries the description: "A very simple reset that sits on top of Normalize.css."

**Comment by nemanjaglumac** [comment]
I understand. 
But, if you think about it, it is still a personal preference and therefore can be a project specific (the way I see it). Just imagine if someone uses serif font as their base font for the whole project and it perfectly suits them to use normalize.css proposed sans serif for buttons etc. 
If we reset normalize.css (override that piece of code), such user would need to then override proposed reset to basically revert it back to original normalize.css. That's two overrides. 

If there's no proposed reset, there would be either 0 or maybe 1 override, depending on your project style and preference. I mean, that's why they say it's opinionated. 

Don't get me wrong - I am just thinking out loud. Does it make sense?

**Comment by florianbouvot** [comment]
@lhermann [normalize.css v8](https://github.com/necolas/normalize.css/blob/master/normalize.css#L157) inherit font-family but we can't update to this version because of browser support...[#353].

However that's a good point and I think we could add it to `_generic.reset.scss`. @inuitcss/core  what do you think ?

**Comment by herzinger** [comment]
@florianbouvot it was decided that the branch for the next release (v7.0.0) would drop legacy browsers support.

**Comment by florianbouvot** [comment]
@herzinger yes I remember but for v6 we could add this font-family reset, what do you think?

**Comment by lhermann** [comment]
Great! I can create a pull request if you like.

**Comment by herzinger** [comment]
I'm of the opinion we shouldn't really touch v6 anymore (and conversely, should release v7 as soon as possible).

**Comment by florianbouvot** [comment]
@herzinger ok, I will work on it soon ;)

**Comment by florianbouvot** [comment]
@lhermann I close this issue because it will be fixed in next version 7.0.0.
